.travis.yml

```diff
@@ -0,0 +1,13 @@
+dist: trusty
+language: cpp
+
+compiler:
+  - gcc
+  - clang
+  
+script:
+  - mkdir build && cd build
+  - cmake -DMATERIALX_BUILD_PYTHON=ON ..
+  - make -j4
```

Comment:
Note: TravisCI gives you 1.5 cores. It's probably not helpful to -j more than 2.